docker/nugus_sim/run.py

```diff
@@ -7,32 +7,40 @@
 
 import ruamel.yaml
 
-BINARIES_DIR = "/home/nubots/NUbots/binaries"
-CONFIG_DIR = BINARIES_DIR + "/config"
+DEFAULT_BINARIES_DIR = "/home/nubots/NUbots/binaries"
+DEFAULT_CONFIG_DIR = DEFAULT_BINARIES_DIR + "/config"
 
-ENV_VARS = ("ROBOCUP_ROBOT_ID", "ROBOCUP_TEAM_COLOR", "ROBOCUP_SIMULATOR_ADDR")
+REQUIRED_ENV_VARS = ("ROBOCUP_ROBOT_ID", "ROBOCUP_TEAM_COLOR", "ROBOCUP_SIMULATOR_ADDR")
+OPTIONAL_ENV_VARS = (
+    "ROBOCUP_TEAM_PLAYER1_IP",
+    "ROBOCUP_TEAM_PLAYER2_IP",
+    "ROBOCUP_TEAM_PLAYER3_IP",
+    "ROBOCUP_TEAM_PLAYER4_IP",
+)
 
 
 def read_args() -> dict:
     parser = argparse.ArgumentParser(description="Run a role for webots RoboCup")
 
     parser.add_argument("role", help="The role to run")
+    parser.add_argument("--goalie", action="store_true", default=False, help="Run the role with goalie behaviour")
```

Comment:
Remove this if we are going to be changing the configs in a different/conflicting way